README.md

```diff
@@ -1,49 +1,185 @@
-Puppeteer will then run an headless Chrome to generate a PDF from the HTML 5 page.
+If installation was successful, the command should report the version of Asciidoctor PDF.
+
+```console
+$ asciidoctor-pdf --version
+Asciidoctor PDF 1.0.0-alpha.3 using Asciidoctor.js 2.0.0 (Asciidoctor 2.0.6) [https://asciidoctor.org]
+Runtime Environment (node v10.15.1 on linux)
+CLI version 3.0.1
+```
+
+## Getting started
+
+Asciidoctor PDF provides a standard document layout.
+To convert an AsciiDoc document using this layout, open a terminal and type:  
+
+    $ asciidoctor-pdf document.adoc
+
+## Configuration
+
+The standard document layout can be configured depending on your needs.
+
+**STEM support**
 
-## Examples
+To activate equation and formula support, set the `stem` attribute in the document's header (or by passing the attribute to the command line):
 
-We provides a few examples in the `examples` directory:
+    $ asciidoctor-pdf document.adoc -a stem
 
-**Document**
+**Title page**
 
-    asciidoctor-pdf ./examples/document/document.adoc
-    asciidoctor-pdf ./examples/document/basic-example.adoc
+The title page is enabled if either of these conditions are met:
 
-**Letter**
+- The document has the `book` doctype.
+- The `title-page` attribute is set (with an empty value) in the document header.
 
-    asciidoctor-pdf ./examples/letter/letter.adoc --template-require ./examples/letter/template.js
 
-**Book**
+    $ asciidoctor-pdf document.adoc -a title-page
+
+**Additional styles**
+
+You can provide a custom stylesheet using the `stylesheet` attribute. 
+You can also specify where the stylesheet is located with the `stylesdir` attribute.
 
-    asciidoctor-pdf ./examples/book/book.adoc --template-require ./examples/book/template.js
+    $ asciidoctor-pdf document.adoc -a stylesheet=custom.css
 
-**Slides**
+**NOTE:** The default stylesheet will still be applied.
+While it's possible to override existing rules, the goal is to provide additional styles for custom roles, for instance:
 
-    asciidoctor-pdf ./examples/slides/slides.adoc --template-require ./examples/slides/template.js
+```adoc
+Please edit the [.path]_package.json_ file.
+```
+
+```css
+.path {
+  color: #fecbcb;
+}
+```
+
+## Custom layout
+
+It's also possible to create your own layout by extending the default HTML 5 converter.
+To create a new layout you will need some JavaScript knowledge.
+
+Let's say that we want to override how the `document` node is converted.
+
+```js
+module.exports = {
+  document: (node) => `<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<link href="./layout.css" rel="stylesheet">
+</head>
+<body>
+${node.getContent()}
+</body>`,
+}
+```
+
+In the above example, we are using [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) but you can use your favorite template engine.
+You can also override other elements. Here's the complete list:
```

Comment:
"Here's the complete list" -> could this be linked to a source? I'd think it might become outdated once the whatever backs it with the code changes...?

Reply:
As far as I know there's no official documentation with the complete list. 
I can link to the source code: https://github.com/asciidoctor/asciidoctor/blob/fc4a66124fbb68277baa111623e988c965b5e02c/lib/asciidoctor/converter/html5.rb#L50-L90 ?

Reply:
Not everyone is familiar with ruby, and the line numbers might change. Therefore I think the list should be in the document as is (despite being a terrifying list as it scrolls for more than a screen's height). And adding the link to the source code (maybe just the file, maybe with reference to the line number) might help once new elements pop up or the list needs to be updated/reviewed.

Reply:
It's not (yet) an official documentation but I could link to the Asciidoctor.js documentation where we explain how to write a custom converter: https://asciidoctor-docs.netlify.com/asciidoctor.js/extend/converter/custom-converter/#custom-converter-class

I should probably assign an id to the paragraph "Here’s the complete list of node’s name".

Reply:
It's a good use case for a `<details><summary>Complete list of elements</summary> ... the list ...</details>`.

If it's a good location but too long, this will fold the content until a user reveals it.

Reply:
It's a good idea, I will update it, thanks 👍